### PR TITLE
Lesson_admin branch merge

### PR DESCRIPTION
**Pull request by bigkeios**
This pull request is for merging these implementations into master:
- Lesson belongs to Course and Course has many Lesson
- Index of lessons in their course view, show view of a lesson
- Admin new/create lesson

### CODE REVIEW

File: app/views/lessons/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<% provide :title, t(:add_lesson) %>
+<% provide :button_text, t(:add_lesson) %>
+<h1> <%= t(:add_lesson) %> </h1>
+
+<div class="row">
+  <div class="col-md-6 offset-md-3">
+      <%= form_for [@course, @lesson] do |f| %>
```
Comment (huannd-0028) [inline app/views/lessons/new.html.erb]: indent

File: app/controllers/lessons_controller.rb [hunk 2]
```diff
@@ -0,0 +1,47 @@
+class LessonsController < ApplicationController
+  include SessionsHelper
+  before_action :logged_in_user, only: %i[new create]
```
Comment (minhdt-0293) [inline app/controllers/lessons_controller.rb]: before_action :logged_in_user, :admin_user, :set_lesson, only: %i(new create)

File: test/controllers/lessons_controller_test.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
+
+class LessonsControllerTest < ActionDispatch::IntegrationTest
+  test "should get new" do
+    get lessons_new_url
+    assert_response :success
+  end
+
```
Comment (huannd-0028) [inline test/controllers/lessons_controller_test.rb]: remove line

**Comment by bigkeios** [comment]
Fixed

**Comment by huannd-0028** [comment]
Ok

**Pull request merged**